bee-network/src/tests/add_peer.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::common::{await_events::*, keys_and_ids::*, network_config::*, shutdown::*};
+
+use crate::{standalone::init, Command, PeerRelation};
+
+#[tokio::test]
+#[serial_test::serial]
+async fn add_peer() {
+    let config1 = get_in_memory_network_config(1337);
+    let keys1 = gen_random_keys();
+
+    let config2 = get_in_memory_network_config(4242);
+    let keys2 = gen_random_keys();
+
+    let network_id = gen_constant_net_id();
+
+    let (tx1, mut rx1) = init(config1, keys1, network_id, shutdown(10)).await;
+    let (_, mut rx2) = init(config2, keys2, network_id, shutdown(10)).await;
+
+    let _peer_id1 = get_local_id(&mut rx1).await;
+    let _address1 = get_bind_address(&mut rx1).await;
+    // println!("(1) Peer Id: {}", peer_id1);
```

Comment:
`println!` :trollface:

Reply:
it's just test code. I'ld like to be able to turn the `println!`s on quickly, so I leave them for now :smiley: